File: TheaterSuggestions/CSharp/TheaterSuggestions.Tests/AuditoriumSeatingArrangement.cs

using System.Collections.Generic;

namespace SeatsSuggestions.Tests;

public class AuditoriumSeatingArrangement
{
    public readonly Dictionary<string, Row> Rows;

    public AuditoriumSeatingArrangement(Dictionary<string, Row> rows)
    {
        Rows = rows;
    }
}

File: TheaterSuggestions/Java/ScreeningSeatingArrangementSuggestionsAcceptanceTests/src/test/java/org/weaveit/seatssuggestionsacceptancetests/AuditoriumSeatingArrangement.java
package org.weaveit.seatssuggestionsacceptancetests;

import java.util.Map;

public class AuditoriumSeatingArrangement {

    private final Map<String, Row> rows;

    public AuditoriumSeatingArrangement(Map<String, Row> rows) {
        this.rows = rows;
    }

    public Map<String, Row> rows() {
        return rows;
    }
}
